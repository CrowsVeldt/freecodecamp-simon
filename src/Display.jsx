import React from 'react'
import styled from 'styled-components'

export default function Display (props) {
  return (

    <StyledDisplay>
      <p>{props.title}</p>
    </StyledDisplay>

  )
}

const StyledDisplay = styled.div`
background-color: black;
color: white;
height: 30px;
width: 60px;
display: flex;
justify-content: center;
align-items: center;
`
