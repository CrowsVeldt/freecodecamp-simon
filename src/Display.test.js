import Display from './Display'
import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

describe ('the Display component', () => {

  test('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Display />, div)
  })

  test('renders correctly', () => {
    const tree = renderer.create(<Display />)
    expect(tree).toMatchSnapshot()
  })

})
