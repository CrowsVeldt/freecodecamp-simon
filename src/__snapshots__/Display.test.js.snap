// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the Display component renders correctly 1`] = `
<div
  className="sc-bdVaJa iuEQjT"
>
  <p />
</div>
`;
